module/zfs/dsl_scan.c

```diff
@@ -141,9 +142,9 @@ dsl_scan_init(dsl_pool_t *dp, uint64_t txg)
 		 */
 		if (err == EOVERFLOW) {
```

Comment:
I think I can avoid modifying the on-disk format (`dsl_scan_phys_t`) here, as `scn_scrub_pause_time` doesn't need to be persistent and that will actually make the code simpler.

However, you do bring up an interesting point - errata 2094 locks us into the current on-disk format for dsl_scan_phys_t...